BraveWallet/Preview Content/MockKeyringService.swift

```diff
@@ -60,81 +65,86 @@ class MockKeyringService: BraveWalletKeyringService {
     let info = BraveWallet.AccountInfo()
     info.name = accountName
     info.address = nextAddress()
-    keyring.accountInfos.append(info)
-    observers.allObjects.forEach {
-      $0.accountsChanged()
+    keyringInfo(BraveWallet.DefaultKeyringId) { [self] keyring in
```

Comment:
All these methods should not call `keyringInfo`, and should access `keyrings` directly. In fact, `keyringInfo` should probably return a copy, not the direct ref to match closer to actual implementation